dpe/src/commands/mod.rs

```diff
@@ -108,6 +108,14 @@ pub trait CommandExecution {
         env: &mut DpeEnv<impl DpeTypes>,
         locality: u32,
     ) -> Result<Response, DpeErrorCode>;
+
+    #[cfg(not(feature = "no-cfi"))]
+    fn __cfi_execute(
+        &self,
+        dpe: &mut DpeInstance,
+        env: &mut DpeEnv<impl DpeTypes>,
+        locality: u32,
+    ) -> Result<Response, DpeErrorCode>;
```

Comment:
nit: Include a comment that this is implemented by the derive trait

Reply:
Done